common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -747,6 +737,21 @@
                     self.isMouseFocus = false;
                     self.rendered = true;
                     self.state.el.addClass('is-captions-rendered');
+
+                    self.subtitlesEl.attr('aria-label', gettext('Video transcript: activating an item in this group will spool the video to the corresponding time point.')) // jshint ignore:line
```

Comment:
@clrux happy to contribute. Could you give me a little more context? I read the PR and JIRA story, and tried to look at both sandboxes (not found), but I'm not clear on what exactly the learner is doing when this message appears.